Metrics/Sources/MetricsLib/UploadMetrics.swift

```diff
@@ -47,30 +47,30 @@ public struct UploadMetrics: Encodable {
 public struct TableUpdate: Encodable {
   public var table_name: String
   public var column_names: [String]
-  public var replace_measurements: [[Double]]
+  public var replace_measurements: [[String]]
 
-  public init(table_name: String, column_names: [String], replace_measurements: [[Double]]) {
+  public init(table_name: String, column_names: [String], replace_measurements: [[String]]) {
     self.table_name = table_name
     self.column_names = column_names
     self.replace_measurements = replace_measurements
   }
 
   /// Creates a table update for code coverage by parsing a coverage report from XCov.
-  public static func createFrom(coverage: CoverageReport, pullRequest: Int) -> TableUpdate {
-    var metrics = [[Double]]()
+  public static func createFrom(coverage: CoverageReport, pullRequest: Int, currentTime: String) -> TableUpdate {
+    var metrics = [[String]]()
     for target in coverage.targets {
-      let sdkKey = TARGET_TO_SDK_INDEX[target.name]
-      if sdkKey == nil {
-        print("WARNING - target \(target.name) has no mapping to an SDK id. Skipping...")
-      } else {
-        var row = [Double]()
-        row.append(Double(pullRequest))
-        row.append(sdkKey!)
-        row.append(target.coverage)
+      if TARGETS_TO_COLLECT.contains(target.name) {
+        var row = [String]()
+        row.append(target.name.components(separatedBy: "_")[0])
+        row.append(String(pullRequest))
+        row.append(String(target.coverage))
+        row.append(currentTime)
         metrics.append(row)
+      } else {
+        print("WARNING - target \(target.name) has no mapping to an SDK id. Skipping...")
```

Comment:
Maybe need to reword this message now that the mapping is gone?

Reply:
+1